backend/streetdrop-api/src/main/java/com/depromeet/common/repository/BannedWordRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.depromeet.common.repository;
+
+import com.depromeet.common.entity.BannedWord;
+import io.lettuce.core.dynamic.annotation.Param;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+
+public interface BannedWordRepository extends JpaRepository<BannedWord, Long> {
+
+    @Query("SELECT bw.word FROM BannedWord bw WHERE bw.word IN :words")
+    List<String> findBannedWordsInWordList(@Param("words") List<String> words);
```

Comment:
현재 단어 1개를 반환하는데, 리턴 값과 메서드 명이 수정되어야 할 것 같아요! 

```suggestion
    @Query("SELECT bw.word FROM BannedWord bw WHERE bw.word IN :words")
    String findBannedWordsInWord(@Param("words") List<String> words);
```

Reply:
IN 으로 여러개가 반환될 수 되있어서 List로 처리하고 있습니다